axelrod/strategies/axelrod_second.py

```diff
@@ -1508,3 +1508,42 @@ def strategy(self, opponent: Player) -> Action:
         return random_choice(self.prob_coop[(recent_history[-3],
                                              recent_history[-2],
                                              recent_history[-1])])
+
+
+class White(Player):
+    """
+    Strategy submitted to Axelrod's second tournament by Edward C Whtie (K72R)
+    and came in thirteenth in that tournament.
+
+    If the opponent Cooperated last turn or in the first ten turns, then
```

Comment:
*  If the opponent cooperated last turn or in the first ten turns, then
   cooperate.
*  Otherwise defect if and only if:
      floor(log(turn)) * opponent.defections >= turn